bwc-test/build.gradle

```diff
@@ -148,24 +163,7 @@ List<Provider<RegularFile>> plugins = [
 // Creates a test cluster with 3 nodes of the old version.
 2.times {i ->
     task "${baseName}#oldVersionClusterTask$i"(type: StandaloneRestIntegTestTask) {
-        exclude '**/*Test*'
-        exclude '**/*Sanity*'
         useCluster testClusters."${baseName}$i"
-        if (System.getProperty("mixedCluster") != null) {
```

Comment:
We do not need any of these tests any more? I know you added coverage in the action but seems like we may still be mixing some of these tests after the changes?

Reply:
This filter logic is really complex and I'm not sure it accomplishes much.   System properties are set indicating what kind of test is running that can be read by the test, e.g. `systemProperty 'tests.rest.bwcsuite', 'old_cluster'`  When looking at our previous BWC tests they didn't do much different between the different kinds of testing.


Created the following issue to figure out what tests we should add:
- https://github.com/opensearch-project/security/issues/2259

Reply:
Perfect! I know you still have some tests, I just wanted to get a read on what you thought about testing moving forward.